src/FSharp.Data.GraphQL.Server.AspNetCore/GraphQLWebsocketMiddleware.fs

```diff
@@ -282,14 +282,10 @@ type GraphQLWebSocketMiddleware<'Root>
     // <-- Main
     // <--------
 
-    let waitForConnectionInitAndRespondToClient
-        (serializerOptions : JsonSerializerOptions)
-        (connectionInitTimeoutInMs : int)
-        (socket : WebSocket)
-        : TaskResult<unit, string> =
-        taskResult {
+    let waitForConnectionInitAndRespondToClient (socket : WebSocket) : TaskResult<unit, string> =
+        task {
             let timerTokenSource = new CancellationTokenSource ()
-            timerTokenSource.CancelAfter (connectionInitTimeoutInMs)
+            timerTokenSource.CancelAfter connectionInitTimeout
```

Comment:
having the suffix `...InMs` is a nice hint: why not have it?

Reply:
ok, I won't die on that hill.

Reply:
I just thought that it is important at the API level but not important detail in the private code